src/main/java/edu/kis/powp/jobs2d/command/visitor/canvas/Canvas.java

```diff
@@ -0,0 +1,13 @@
+package edu.kis.powp.jobs2d.command.visitor.canvas;
+
+public abstract class Canvas {
+	protected int width;
+	protected int height;
+
+	public Canvas(int width, int height) {
+		this.width = width;
+		this.height = height;
+	}
+
```

Comment:
This is a RectangularCanvas. Please extract also a sole interface to enable support of wider range of canvases.